Web/hermes.client-web/src/store/modules/auth.js

```diff
@@ -0,0 +1,64 @@
+import AuthService from '../../services/auth.service';
+
+ export const auth = {
+   namespaced: true,
+  state:  {
+    status: { loggedIn: false }
+  },
+  actions: {
+    attemptLogin({ commit }, {email,password}) {
+      return AuthService.login({email,password})
+      .then(
+        response => {
+          commit('loginSuccess', response);
+
+          return Promise.resolve(response);
+        },
+        error => {
+          commit('loginFailure');
+          return Promise.reject(error);
+        }
+      );
```

Comment:
An thes to `attemptLogin` na gyrnaei kai ayto `promise` tote:
```suggestion
      return new Promise((resolve, reject) => {
         AuthService.login({email,password})
         .then(response => {
             commit('loginSuccess', response);
             resolve(response);
         }).catch((err) => {
             console.log(err);
             commit('loginFailure');
             reject(err);
         });
      })
```